study.md

```diff
@@ -31,7 +31,11 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+practice_array = [23, 56, 3, 7]
+  practice_array.reverse!
```

Comment:
This reverses the array in place but does not sort it beforehand.  `practice_array.sort!.reverse!`.